public/hooks/use_get_chunks_from_http_response.tsx

```diff
@@ -18,7 +18,18 @@ export const useGetChunksFromHTTPResponse = () => {
     abortController: AbortController;
   }) => {
     const chunk$ = new BehaviorSubject<StreamChunk | undefined>(undefined);
-    const messageContentPuller = new MessageContentPuller();
+    const messageContentPuller = new MessageContentPuller({
+      isContentReadyToUse(message) {
+        const regexpForHyperlink = /\[([^\]\n]+)\]\(([^)\n]+)\)/;
+
+        // For message contains [, but not completely comply with hyperlink format
+        if (message.includes('[') && !regexpForHyperlink.test(message)) {
+          return false;
+        }
+
+        return true;
+      },
+    });
```

Comment:
It's nice we have this logic to improve rendering experience. My concern is markdown itself support lots of format like table, it will render differently once it reached to its end tag. I would prefer not add those special logic.

Also, if the output stoped at the middle of a hyperlink due to max output token limit reached, what will displayed to user?

Reply:
In that case, we will display [hyperlink](http://www.link-not-closed.

> I would prefer not add those kind of special logics.
Yeah, the hyperlink experience is bad so we have to address it. Actually I tried for table format, and that works fine without properly closed. 
![image](https://github.com/user-attachments/assets/9e056395-4db5-422f-b677-53eb0129dee7)